src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/logs/parsers/PlayerIPParser.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.quiltmc.community.modes.quilt.extensions.logs.parsers
+
+private const val IP_REGEX =
+    "[a-zA-Z]+\\[/[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+:[0-9]+] logged in with entity id"
+
+class PlayerIPParser : BaseLogParser {
+    override suspend fun getMessages(logContent: String): List<String> {
+        val messages: MutableList<String> = mutableListOf()
+
+        if (logContent.contains(IP_REGEX)) {
+            @Suppress("MaximumLineLength")
+            messages.add(
+                "You appear to have shared player IP addresses. " +
```

Comment:
I'd prefer something more in line with the messages already there. For example:

`This log appears to contain players' IP addresses - please consider deleting your uploaded log, and posting it again with the IP addresses removed`